client/src/main/scala/org/http4s/client/Client.scala

```diff
@@ -192,4 +200,50 @@ final case class Client(open: Service[Request, DisposableResponse], shutdown: Ta
     shutdown.run
 }
 
+<<<<<<< HEAD
+object Client {
+  /** Creates a mock client from the specified service.  Useful for generating
+    * pre-determined responses for requests in testing.
+    * 
+    * @param service the service to respond to requests to this client
+    */
+  def mock(service: HttpService): Client = {
```

Comment:
I must admit that I'm being an armchair architect here, but in my mind's eye the test kit is where the integrations live, then they are useful for others as well.
I'm not firmly opposed to test related stuff being in the main project, but it doesn't feel like it belongs there. If it is in the main project, I think a different package would be better.